flycheck-haskell.el

```diff
@@ -143,6 +143,9 @@ Return the configuration."
   (or (flycheck-haskell-get-cached-configuration cabal-file)
       (flycheck-haskell-read-and-cache-configuration cabal-file)))
 
+(defconst flycheck-haskell-cabal-config "cabal.config"
```

Comment:
@rpglover64 Ok, you're the expert, you make the choices.  As a matter of fact, I didn't even know that `cabal.config` existed until you made this PR ;)